crates/katana/runner/src/deployer.rs

```diff
@@ -0,0 +1,50 @@
+use anyhow::{Context, Ok, Result};
+use starknet::core::types::FieldElement;
+use tokio::process::Command;
+
+use crate::KatanaRunner;
+
+impl KatanaRunner {
+    /// Known issue - rpc set in Scarb.toml overrides command line argument
+    pub async fn deploy(&self, manifest: &str, script: &str) -> Result<FieldElement> {
+        let rpc_url = &format!("http://localhost:{}", self.port);
+
+        let out = Command::new("sozo")
```

Comment:
hm im not sure if this should live in the katana crate

Reply:
moved to the benches crate